web-client/package.json

```diff
@@ -12,7 +12,7 @@
     "serve17": "export NODE_OPTIONS=--openssl-legacy-provider && vue-cli-service serve"
   },
   "dependencies": {
-    "@epfml/discojs": "file:../discojs/discojs-web",
+    "@epfml/discojs": "^2.1.1",
```

Comment:
Made this change according to what Hugo said in the onboarding video at 14:00, package.json should not be pointing to the local file.

Reply:
yep, it won't matter anymore with the use of #617 as `npm` takes care of that for us